### DISCUSSION TITLE
Unauthenticated

### DISCUSSION THREAD

**Discussion by bacipher** [getcandy/api-boilerplate#2] [Help]
This is good. I really want to use this project using sanctum and I have done all the configuration and debugging but I get unauthorized error. Maybe it's because I was configuring from scratch and was using fortify. Will try installing this boilerplate and see if it will speed up my consumption of the API
Thanks for this

**Comment by bacipher** [comment]
Or is there any recommendations while using fortify? Would highly appreciate

**Comment by glennjacobs** [comment]
I don't think it will be related to Fortify, check the troubleshooting here https://docs.getcandy.io/hub/getting-started/troubleshooting.html

Or jump on Discord and we'll try help.
